WalletWasabi/Helpers/Constants.cs

```diff
@@ -22,6 +22,13 @@ public static class Constants
 			MinProtocolCapabilities = new ProtocolCapabilities { SupportGetBlock = true, SupportWitness = true, SupportMempoolQuery = true }
 		};
 
+		public static readonly NodeRequirement LocalNodeRequirements = new NodeRequirement
+		{
+			RequiredServices = NodeServices.NODE_WITNESS,
+			MinVersion = ProtocolVersion_WITNESS_VERSION,
+			MinProtocolCapabilities = new ProtocolCapabilities { SupportGetBlock = true, SupportWitness = true }
```

Comment:
Does SupportGetBlock work with pruned node?

Reply:
What a node can (and cannot) do depends on two different this: the `protocol version` and the `signaled services`. In this case the `ProtocolCapabilities` class is used to check the requirements against the peers protocol version.

If the protocol version supports get full node the we have to check if the peer provides the service `NETWORK` (what means it can provides the full blockchain) or `NODE_NETWORK_LIMITED` (what means it can provides at least the latest 288 blocks).

Going back to your question, the SupportGetBlock only indicates that the peer protocol version supports that, it doesn't mean it can provides the full blockchain or that it is prunned neither.

Reply:
Good. I can go ahead and do the final review and merge?